codility_training/lessons.lesson06.Sorting.Distinct/sjjyy.java

```diff
@@ -0,0 +1,26 @@
+package lessons.lesson06.Sorting.Distinct;
+
+import java.util.Set;
+import java.util.HashSet;
+
+class sjjyy
+{
+	 public static int solution(int[] A)
+	 {
+		 Set<Integer> hs = new HashSet<>();
```

Comment:
```suggestion
		 Set<Integer> hs = new HashSet(Arrays.stream(A).boxed().boxed().collect(Collectors.toList()));
```
이렇게 하면 11 ~ 15 line 생략 가능합니다.

Reply:
String 형태의 배열의 경우 다음과 같이 Collection으로 변경할 수 있습니다.
``` java
Arrays.asList(arr);
```